assets/charts/components/prometheus-operator/charts/kube-state-metrics/Chart.yaml

```diff
@@ -17,4 +17,4 @@ maintainers:
 name: kube-state-metrics
```

Comment:
Yep anything and everything that has changed under `assets/charts/components/prometheus-operator` is from upstream. If anything otherwise then it accompanies comment which highlights the fact that the change was made by us.